### DISCUSSION TITLE
I have potential sources for regions / sector of France

### DISCUSSION THREAD

**Discussion by Mayakatana915** [electricitymaps/electricitymaps-contrib#5081] [Ideas] [answered]
I need further instructions on how to improve my contribution but here hare some links : 
https://www.rte-france.com/eco2mix
https://trackmywatt.fr

**Comment by VIKTORVAV99** [comment, marked as answer]
Hello!
We are already in the process of using eco2mix data for France in #4853 but because we already have a high quality data source for France, we need to make sure the new source has the same level of quality and reliability before we can make the switch.

As for trackmywatt I don't think it offers anything that isn't already provided by eco2mix or ENTSO-E, so we won't be using that source. 

Hopefully this gives you some clarity around the issue.
